lib/graylog2_exceptions.rb

```diff
@@ -10,7 +10,9 @@ class Graylog2Exceptions
   attr_writer :env_ref
 
   FULL_MESSAGE_FIELDS = %w(HTTP_ORIGIN HTTP_REFERER CONTENT_TYPE HTTP_USER_AGENT REMOTE_ADDR REQUEST_URI FIVERR_MESSSAGE).freeze
-  BACKTRACE_DEPTH = 2
+  BACKTRACE_START = 3
```

Comment:
make sure this is the correct value (3/4?)

Reply:
should be 4